aea/cli/scaffold.py

```diff
@@ -55,6 +55,23 @@
 
 
 @click.group()
+@click.option(
+    "-vendor",
+    default="open_aea",
+    type=str,
+    help="Specify component vendor.",
+)
```

Comment:
Please remove, the vendor should be the default author which is already set on global cli config. (It gets set during `aea init`)